### PR TITLE
Fixed value of columns not updating in ui_bookmark table

### PR DESCRIPTION
**Pull request by gelanivishal**
### Description (*)
There was data type issue in ui_bookmark table. 

### Fixed Issues (if relevant)

1. magento/magento2#18557: Value of created_at and updated_at columns not updating in ui_bookmark table

### Manual testing scenarios (*)

1. Check phpmyadmin and go to **ui_bookmark** table

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @gelanivishal. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me $VERSION instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

File: app/code/Magento/Ui/Setup/UpgradeSchema.php [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+     * @SuppressWarnings(PHPMD.ExcessiveMethodLength)
+     * @SuppressWarnings(PHPMD.CyclomaticComplexity)
+     * @SuppressWarnings(PHPMD.NPathComplexity)
+     */
+    public function upgrade(SchemaSetupInterface $setup, ModuleContextInterface $context)
+    {
+        $setup->startSetup();
+
+        if (version_compare($context->getVersion(), '2.0.1', '<')) {
+            $setup->getConnection()->dropColumn($setup->getTable('ui_bookmark'), 'created_at');
```
Comment (slavvka) [inline app/code/Magento/Ui/Setup/UpgradeSchema.php]: Could you please tell me why you drop and recreate the fields instead of updating them?

**Review by slavvka — CHANGES_REQUESTED** [review]

File: app/code/Magento/Ui/Setup/UpgradeSchema.php (hunk 1 above)
Comment (gelanivishal) [reply app/code/Magento/Ui/Setup/UpgradeSchema.php]: Please give me sample

Comment (slavvka) [reply app/code/Magento/Ui/Setup/UpgradeSchema.php]: E.g. using `\Magento\Framework\DB\Adapter\AdapterInterface::modifyColumn` method

**Comment by slavvka** [comment]
@gelanivishal Thank you for the changes. We want all PRs to be delivered to 2.3 and than backported to 2.2. Could you please port it to 2.3 and then we will accept this one? Please pay your attention that we introduced Declarative Schema in 2.3 so you need to implement the fix in different way for 2.3. Please refer to the [guide](https://devdocs.magento.com/guides/v2.3/extension-dev-guide/declarative-schema/)

**Comment by gelanivishal** [comment]
@slavvka GREAT!! You can tell me at once comment. I have to create 3 PR for the same issue.

**Comment by slavvka** [comment]
@gelanivishal Sorry but we need to make sure that changes in 2.2 will exist in 2.3 as well to maintain cross-versioned integrity. Does it make sense?

**Comment by sidolov** [comment]
Hi @gelanivishal , I am closing this PR now due to inactivity.
Please reopen and update if you wish to continue.
Thank you for the collaboration!

**Comment by gelanivishal** [comment]
@sidolov Please check changes. Magento 2.3 and 2.2 don't have same changes.

**Comment by slavvka** [comment]
Hey @gelanivishal. This PR was closed due to inactivity. Please fell free to proceed with this PR once you port it to 2.3

**Comment by gelanivishal** [comment]
@slavvka It's already open. Can you please review my changes?

**Comment by slavvka** [comment]
Hey @gelanivishal. This PR is fine. But we want to have all PRs to be delivered to 2.3 firstly. Please create a upport to 2.3 and we will process it. I already mentioned before how to create PRs to 2.3 since there was introduced the declarative schema. thank you!

**Comment by slavvka** [comment]
Hey @gelanivishal I am closing this PR due to inactivity. Please feel free to re-open it once you up-port it to 2.3 (with appropriate approach). Thank you!

**Pull request closed without merging**